tests/foreman/ui/test_gpgkey.py

```diff
@@ -31,11 +31,12 @@
 from robottelo.ui.session import Session
 from robottelo.vm import VirtualMachine
 
+GEN_STRING_LIST_ARGS = dict(exclude_types=['numeric'], bug_id=1184480)
```

Comment:
:+1: but I would use dict literal notation. Personally I don't like using dict constructor since it has more overhead when creating a dict. This is not a blocker since this operation is done one time.

I'm just saying :smile: